module/Motion/CorrectingRotation.cpp

```diff
@@ -0,0 +1,153 @@
+/**
+ * @file   CorrectingRotation.h
+ * @brief  角度補正回頭
+ * @author aridome222
+ */
+
+#include "CorrectingRotation.h"
+
+using namespace std;
+
+CorrectingRotation::CorrectingRotation(int _targetAngle, double _targetSpeed)
+  : targetAngle(_targetAngle), targetSpeed(_targetSpeed)
+{
+}
```

Comment:
```suggestion
{};
```
セミコロン抜け。他ファイルはあったと思うので一応。
セミコロンなくても Google Test も build も通った記憶はあります。セミコロンあるなしで動作に影響が出ることってさすがにないですよね？　コメントしておいてなんですが、ないならわざわざ直すほどでもない気はします。